services/app-api/src/resolvers/createHealthPlanPackage.ts

```diff
@@ -41,10 +41,7 @@ export function createHealthPlanPackageResolver(
             input.programIDs
         )
 
-        if (
-            programs === undefined ||
-            programs.length !== input.programIDs.length
-        ) {
+        if (programs instanceof Error) {
```

Comment:
I thought the plan was just to pass _all_ of the programs for a given state into the emailer and have it be responsible for the parsing. then we skip dealing with this error all together.